lib/active_admin/async_exporter/reports/worker.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'pry'
+
```

Comment:
```suggestion
```

Reply:
Hmm, kind of weird, but when I remove that require and try to run `rspec` this happens:
![image](https://user-images.githubusercontent.com/42209955/96295067-b1258e80-0fc3-11eb-8842-62face156696.png)

Any ideas? 😬

Reply:
This means that `pry` is a runtime dependency then. We should require it in the gem entry file, and not only in the specs. No?

Reply:
Okay, adding the require on spec/dummy/config/application.rb worked, commiting!